packages/components/RadioGroup/src/RadioButton.settings.ts

```diff
@@ -61,23 +62,22 @@ export const settings: IComposeSettings<IRadioButtonType> = [
       style: {
         marginTop: 3,
         borderStyle: 'dashed',
-        borderColor: 'transparent',
         borderWidth: 1
       }
     },
-    // This still uses the innerCircle's 'hover' style instead of the 'focused' style when need be. I'm commenting out'
-    _precedence: ['disabled', 'hovered', 'selected'],
+    // This still uses the innerCircle's 'hover' style instead of the 'focused' style when need be. I'm commenting out
```

Comment:
I don't understand what this comment means, could you clarify?

Reply:
That looks like an old comment that isn't relevant anymore. Good catch, I'm removing it now